database/factories/CountryFactory.php

```diff
@@ -15,7 +15,7 @@ public function definition(): array
             "alternative_name" => fake()->unique()->city(),
             "latitude" => fake()->latitude(),
             "longitude" => fake()->longitude(),
-            "iso" => fake()->unique()->languageCode(),
+            "iso" => fake()->unique()->languageCode() . "x",
```

Comment:
x?

Reply:
"Real" countries are from CountrySeeder and have iso field unique so "fake" countries get "x" to prevent unique error.